### ISSUE TITLE
a wee bit of purification needed

### ISSUE THREAD

**Issue by sydb** [TEIC/TEI#1464]
As posted to Council list yesterday, and affirmed by @lb42:

1. The only example of `<macroRef>` in the Guidelines incorrectly refers to an old `data.*` datatype instead of an actual macro.

1. The content of `<att>` and `<gi>` is incorrectly specified with `<macroRef>` instead of `<dataRef>`

1. The first example of `<dataRef>` in its tagdoc refers to an old style datatype. Given that the `@source` is `"1.5.0"` it is technically correct, but exemplifies an old way of doing things.

1. I also discovered (after the post to Council list) that the specification of `teidata.percentage` is missing its `@restriction`.

Note: this was erroneously [Stylesheets issue 156](https://github.com/TEIC/Stylesheets/issues/156) for a short while. Sorry.

**Comment by sydb** [comment]
These have all been addressed in 30db34a; I will close this ticket if and when Mr. Jenkins is happy with the result.

**Comment by sydb** [comment]
No, Mr. Jenkins is **not** happy. Looks to me like the Stylesheets knew to generate "#PCDATA" from `data.name` inside content, but do not know to do that with `teidata.name` inside content, and so we end up with invalid DTDs that have 
~~~~~
<!ELEMENT gi %teidata.name;>
~~~~~
instead of
~~~~~
<!ELEMENT gi  (#PCDATA)>
~~~~~
The former boils down to
~~~~~
<!ELEMENT gi ' CDATA '>
~~~~~
which, of course, is not valid in a DTD.

However, this is clearly a Stylesheets issue, not a Guidelines issue, so I am going to close this and open a ticket against the Stylesheets.

**Comment by lb42** [comment]
This is Not Good. Maybe consider reverting the changes to content model of &lt;gi> and &lt;att> till we can sort this out properly?

**Comment by sydb** [comment]
If necessary, absolutely. But if we can leave it (build process) broken for awhile, it serves as good incentive to fix this problem. :-)

**Comment by hcayless** [comment]
If it's a Stylesheets bug and I need to fix it, it'll stay broken until I'm back from vacation :-).

**Comment by sydb** [comment]
Fix to [Stylesheets/ issue #157](https://github.com/TEIC/Stylesheets/issues/157) fixed this problem, so closing ticket.
Oh. It’s already closed. :-)
